### PR TITLE
feat: add experience version to data upload (#358)

### PR DESCRIPTION
**Pull request by valentinoli**
#358

### CODE REVIEW

File: components/unit/consent-form/UnitConsentForm.vue [hunk 1]
```diff
@@ -91,7 +91,7 @@ import { createObjectURL, mimeTypes } from '@/utils/utils'
 
 // In the case of changes that would break the import, this version number must be incremented
 // and the function versionCompatibilityHandler of import.vue must be able to handle previous versions.
-const VERSION = 3
+// const VERSION = 3
```
Comment (valentinoli) [inline components/unit/consent-form/UnitConsentForm.vue:94]: I guess we can erase this

**Comment by valentinoli** [comment]
Note: new deployments will break until this PR is merged

File: components/unit/consent-form/UnitConsentForm.vue (hunk 1 above)
Comment (fquellec) [reply components/unit/consent-form/UnitConsentForm.vue:94]: If you delete the if in versionCompatibilityHandler() I think so too

**Review by fquellec — APPROVED** [review]

**Comment by valentinoli** [comment]
Related commit in the monorepo: https://github.com/hestiaAI/hestialabs/commit/5ab69c11f4ccfc1e350c130ae20e5c3241182ce7

**Pull request merged**